lib/minify-inline.js

```diff
@@ -0,0 +1,35 @@
+const terser = require('terser')
```

Comment:
can you stick a `'use strict'` at the top?

Reply:
Fixed